### PR TITLE
Vibrator Deconstruction Recipe

### PR DESCRIPTION
**Pull request by pisskop**
Vibrators can be deconstructed into:
-   [[ "cable", 4 ]],
-   [[ "scrap", 1 ]],
-   [[ "plastic_chunk", 1 ]],

And require an fabrication of 2 and a hacksaw and scredriver.

The recipe is based on the heavy flashlight recipe and adds some scrap metal, a spring, and some plastic chunks.

### CODE REVIEW

**Comment by mugling** [comment]
> [[ "material_aluminium_ingot", 1 ]],

I'd drop this

**Comment by Coolthulhu** [comment]
That's a lot of scrap from something with volume 2. Check out how much volume does this stuff have in total.

**Comment by LlubNek** [comment]
I'd go for a tiny electric motor, a plastic chunk, a chunk of metal and whatever batteries were in it.  A real vibrator wouldn't have anything more than that except a tiny bit of wire and a switch.

**Comment by Cyrano7** [comment]
I would go with just plastic chunks and e_scrap even. Maybe some metal. I can't see there being much useful components in there.

**Comment by pisskop** [comment]
I just want something for it.  Has anyone ever taken apart one?

**Comment by Cyrano7** [comment]
Just fyi there is a electronic scrap item. Its item id is "e_scrap" might be a better fit than having it have a amplifier. Up to you.

**Comment by kevingranade** [comment]
Yes, I disassembled a few broken vibrators, the lowest common denominator
is a motor and flywheel (not sure the flywheel is worth dropping, it's only
a few ounces), a battery compartment, a tiny number of electrical
components (a few resistors and a switch), a miniscule about of structural
plastic (just the handle), and the body, which is silicone.
More complex models have several additional motors attached to some tiny
moving parts (the moving parts are definitely too small to bother with),
and a few more electrical components  ( more switches, some power
transistors, maybe an IC)

Tl;dr I'd say a motor and one unit of e-waste, maybe a second motor. The
rest is too small to bother with unless we had silicone crafting.

**Comment by kevingranade** [comment]
Also, the motor is probably smaller than anything we have in the game, so
that would probably just turn into another unit of e-waste.

**Comment by pisskop** [comment]
[[ "cable", 4 ]],
[[ "scrap", 1 ]],
[[ "plastic_chunk", 1 ]],

Cable is copper wire.  e_scrap could be swapped for it?

**Comment by Cyrano7** [comment]
Ya sounds good to me.

File: data/json/recipes/recipe_deconstruction.json [hunk 1]
```diff
@@ -4183,5 +4183,26 @@
...
+    "difficulty": 3,
+    "time": 5400,
+    "reversible": true,
+    "autolearn": true,
+    "qualities" : [
+    {"id":"SCREW","level":1,"amount":1},
+    {"id":"SAW_M","level":1,"amount":1}
+    ],
+    "components": [
+    [[ "_scrap", 1 ]],
```
Comment (illi-kun) [inline data/json/recipes/recipe_deconstruction.json]: Did you mean ```e_scrap```?
